tests/test_expect.py

```diff
@@ -433,7 +433,7 @@ def test_before_after_exact(self):
         self._before_after(p)
 
     def _ordering(self, p):
-        p.timeout = 5
+        p.timeout = 20
```

Comment:
I got exactly the same false-positive failure again on #313. Please keep this change instead of reverting it!